package.json

```diff
@@ -33,21 +33,21 @@
     "@emotion/babel-preset-css-prop": "^10.0.27",
     "@emotion/core": "^10.0.35",
     "@emotion/styled": "^10.0.27",
-    "@guardian/atoms-rendering": "1.12.0",
-    "@guardian/apps-rendering-api-models": "^0.11.0",
-    "@guardian/bridget": "^0.67",
-    "@guardian/content-api-models": "^15.9.2",
-    "@guardian/content-atom-model": "^3.2.4",
+    "@guardian/apps-rendering-api-models": "^0.10.0",
```

Comment:
I think downgrading some of these libraries might cause problems. Can we stay on the latest versions?

Reply:
Yes it was  bad merge and conflict unresolved, gone through and resolved